### PR TITLE
AiSD-1-fibonacci

### PR DESCRIPTION
**Pull request by mztezi**

### CODE REVIEW

File: fibonacci.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    return output;
+}
+
+function fibonacciI(n) {
+    if (n <= 2) {
+        return n - 1;
+    }
+
+    let output = 0;
```
Comment (kdryzek) [inline fibonacci.js]: use 2 variables only

Comment (mztezi) [reply fibonacci.js]: fixed

File: fibonacci.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }, []);
+
+    return output;
+}
+
+function fibonacciR(n) {
+    if (n <= 2) {
+        output = [...Array(n).keys()];
+    }
+
```
Comment (kdryzek) [inline fibonacci.js]: no empty space here

**Review by kdryzek — APPROVED** [review]

File: fibonacci.js (hunk 2 above)
Comment (mztezi) [reply fibonacci.js]: fixed

**Pull request merged**